example/spec/helpers/testUtils.js

```diff
@@ -50,26 +50,46 @@ function templateFile({ inputTemplateFilename, config }) {
  * Upload a file from the test-data package to the S3 test data
  * and update contents with replacements
  *
- * @param {string} file - filename of data to upload
- * @param {string} bucket - bucket to upload to
- * @param {string} prefix - S3 folder prefix
- * @param {Array<StringReplacement>} [replacements] - array of replacements in file content e.g. [{old: 'test', new: 'newTest' }]
+ * @param {Object} params                         - parameters
+ * @param {string} params.file                    - filename of data to upload
+ * @param {string} params.bucket                  - bucket to upload to
+ * @param {string} params.prefix                  - S3 folder prefix
+ * @param {string} params.targetReplacementRegex  - regexp to allow file copy target to target
+ *                                                  a different s3 key from the original file
+ *                                                  if specified will replace matches with
+ *                                                  targetReplacementString
```

Comment:
Hm this description is a bit confusing to me 🤔  Did you mean to put something like:
```
regexp to allow file copy target to target
a different s3 key from the original file
if specified 
```
?

Reply:
Yeah, thanks, that *was* a bit mangled.

Reply:
e4bc5ad